pom.xml

```diff
@@ -830,6 +830,26 @@
         <scope>import</scope>
         <type>pom</type>
       </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-api</artifactId>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>org.junit.jupiter</groupId>
+        <artifactId>junit-jupiter-engine</artifactId>
+        <scope>test</scope>
```

Comment:
I think maybe this should be in `<dependencies>` not in `<dependencyManagement>`

Reply:
No, they should all be in `dependencyManagement`, to configure the global behavior of these dependencies.